src/uct/rocm/ipc/rocm_ipc_iface.c

```diff
@@ -87,11 +107,11 @@ static ucs_status_t uct_rocm_ipc_iface_query(uct_iface_h tl_iface,
                                           UCT_IFACE_FLAG_PENDING   |
                                           UCT_IFACE_FLAG_CONNECT_TO_IFACE;
 
-    /* TODO: get accurate info */
-    iface_attr->latency                 = ucs_linear_func_make(80e-9, 0);
-    iface_attr->bandwidth.dedicated     = 10.0 * UCS_GBYTE; /* 10 GB */
-    iface_attr->bandwidth.shared        = 0;
-    iface_attr->overhead                = 0.4e-6; /* 0.4 us */
+    iface_attr->latency                 = ucs_linear_func_make(1e-9, 0);
```

Comment:
1 nanosec seems too small?

Reply:
Was not sure how this value impacts protocol selection so decided to use the same value as https://github.com/openucx/ucx/blob/master/src/uct/cuda/cuda_ipc/cuda_ipc_iface.c#L154